crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -180,13 +157,18 @@ impl<P: PhaseItem> DrawFunctions<P> {
 /// );
 /// ```
 pub trait RenderCommand<P: PhaseItem> {
-    /// Specifies all ECS data required by [`RenderCommand::render`].
+    /// Specifies the general ECS data (e.g. resources) required by [`RenderCommand::render`].
     /// All parameters have to be read only.
     type Param: SystemParam + 'static;
+    /// Specifies the ECS data of the view required by [`RenderCommand::render`].
+    /// All components have to be accessed read only.
     type ViewWorldQuery: ReadOnlyWorldQuery;
+    /// Specifies the ECS data of the item required by [`RenderCommand::render`].
+    /// All components have to be accessed read only.
     type ItemWorldQuery: ReadOnlyWorldQuery;
```

Comment:
There should probably be an explanation somewhere of what is a View and an Item in this context.

Reply:
Great idea.
I will take care of that tomorrow as well.

Reply:
Done.